### PR TITLE
Create Python stress test

### PR DESCRIPTION
**Pull request by kpayson64**
Python stress test.   #5986

@sreecha

### CODE REVIEW

File: src/python/grpcio/tests/stress/client.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    name, weight = test_case_arg.split(':', 1)
+    test_case = _test_case_from_arg(name)
+    weighted_test_cases[test_case] = int(weight)
+  return weighted_test_cases
+
+
+def run_test(args):
+  test_cases = _parse_weighted_test_cases(args.test_cases)
+  test_servers = args.server_addresses.split(',')
+  # Propegate any client exceptions with a queue
```
Comment (jtattermusch) [inline src/python/grpcio/tests/stress/client.py]: nit: Propegate/propagate/

**Comment by jtattermusch** [comment]
I took a quick look and things seem alright to me. @sreecha, can you please finish the review?

File: src/python/grpcio/tests/stress/client.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      default='localhost:8080', type=str)
+  parser.add_argument(
+      '--test_cases',
+      help='comma seperated list of testcase:weighting of tests to run',
+      default='large_unary:100',
+      type=str)
+  parser.add_argument(
+      '--test_duration_secs',
+      help='number of seconds to run the stress test',
+      type=int, required=True)
```
Comment (sreecha) [inline src/python/grpcio/tests/stress/client.py]: Default should be -1 and it is not `required`

**Comment by sreecha** [comment]
Except for my one suggestion (to make `test_duration_secs`  `-1` by default), this LGTM!

**Comment by kpayson64** [comment]
Made the suggested feedback changes.

**Comment by sreecha** [comment]
LGTM.

**Comment by sreecha** [comment]
known flakes in Basic tests. Ready to merge.

**Pull request merged**